collect_app/src/main/java/org/odk/collect/android/listeners/SwipeHandler.java

```diff
@@ -96,10 +97,10 @@ public boolean onFling(MotionEvent e1, MotionEvent e2, float velocityX, float ve
                 // Looks for user swipes. If the user has swiped, move to the appropriate screen.
 
                 // For all screens a swipe is left/right of at least .25" and up/down of less than .25" OR left/right of > .5"
-                int xpixellimit = (int) (ScreenUtils.xdpi(odkView.getContext()) * .25);
```

Comment:
A few tests were failing here due to `odkView` being null when we swipe back from the end screen. I'm not sure how we've got away with `SwipeHandler` not dealing with `EmptyView` and `FormEndView` for so long.